core/templates/pages/blog-dashboard-page/modal-templates/upload-blog-post-thumbnail.component.ts

```diff
@@ -94,8 +94,15 @@ export class UploadBlogPostThumbnailComponent implements OnInit {
         this.uploadedImage = decodeURIComponent(
           (e.target as FileReader).result as string);
       }
-      this.changeDetectorRef.detectChanges();
-      this.initializeCropper();
+      try {
+        this.changeDetectorRef.detectChanges();
```

Comment:
Instead of a try-catch, is there a way to check if the view still exists and only then trigger detectChanges?

Reply:
Initially `if (!(this.changeDetectorRef as ViewRef).destroyed)` was used to check if the view exists or not in the edit-profile-picture-modal.component before calling `detectChanges`. However, the flake still occurred. We can see this in PR #14088 where `if (!(this.changeDetectorRef as ViewRef).destroyed)` was changed to `try-catch` to fix the flake.

Reply:
Ah, interesting.. thanks!